From d685205abcb4a51c0a398d793e13d1f64300f15e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Sun, 21 Nov 2021 17:02:50 +0100
Subject: [PATCH] arm64: dts: rk3399-pinebook-pro: Remove unused features

I don't use PCIe on Pinebook Pro and having it enabled just produces
probe failures and delays.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index eb9c69f16890f7..3c1ca3ff51030c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -451,10 +451,6 @@
 	status = "okay";
 };
 
-&hdmi_sound {
-	status = "okay";
-};
-
 &i2c0 {
 	clock-frequency = <400000>;
 	i2c-scl-falling-time-ns = <4>;
@@ -808,7 +804,7 @@
 };
 
 &pcie_phy {
-	status = "okay";
+	//status = "okay";
 };
 
 &pcie0 {
@@ -820,7 +816,7 @@
 	vpcie0v9-supply = <&vcca0v9_s3>;
 	vpcie1v8-supply = <&vcca1v8_s3>;
 	vpcie3v3-supply = <&vcc3v3_ssd>;
-	status = "okay";
+	//status = "okay";
 };
 
 &pinctrl {
